wgpu-hal/src/gles/adapter.rs

```diff
@@ -273,6 +252,27 @@ impl super::Adapter {
             }
         }
 
+        let shading_language_version = {
```

Comment:
Moving this back down is fine - could we just move the "supported GL extensions" printout below this as well?

I was trying to keep all the version numbers together before this massive extension list is printed.